### PR TITLE
Switch to dep from glide

### LINKED ISSUE #52: switch from glide to dep
most of the stack has made this switch

### PR DESCRIPTION
**Pull request by zmanian**
Closes #52

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    "leveldb/memdb",
+    "leveldb/opt",
+    "leveldb/storage",
+    "leveldb/table",
+    "leveldb/util"
+  ]
+  revision = "34011bf325bce385408353a30b101fe5e923eb6e"
+
+[[projects]]
+  branch = "master"
```
Comment (Liamsi) [inline Gopkg.lock]: Nitpick: for other tendermint projects we use the `develop` branch. I wonder if we should stay consistent here.

File: Gopkg.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
```
Comment (Liamsi) [inline Gopkg.toml]: Nitpick: maybe delete this example doc?

File: Gopkg.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "1.1.4"
+
+[[constraint]]
+  branch = "master"
```
Comment (Liamsi) [inline Gopkg.toml]: like above: s/master/develop ?

Comment (zmanian) [reply Gopkg.toml]: Fix this after we figure how how we want integrate amino.

**Comment by ebuchman** [comment]
rebased and merged to develop

File: Gopkg.toml (hunk 3 above)
Comment (ebuchman) [reply Gopkg.toml]: actually best to use versions until/unless we need something from a branch, and then in that situation ideally use a tag

**Pull request closed without merging**